odk1-src/form-logic.rst

```diff
@@ -527,25 +531,13 @@ Repeating groups of questions
 
   If repeats are needed, consider adding some summary calculations at the end so that analysis will not require joining the repeats with their parent records. For example, if you are gathering household information and would like to compute the total number of households visited across all enumerators, add a calculation after the repeats that counts the repetitions in each submission.
 
-To repeat questions or groups of questions
-use the :tc:`begin_repeat...end_repeat` syntax.
-
-.. rubric:: XLSForm --- Single question repeat group
+.. rubric:: XLSForm --- Repeating one or more questions
```

Comment:
I think this should come right after the text above the `seealso` or maybe have it's own blurb?